compiler/front/msgs.nim

```diff
@@ -600,18 +606,20 @@ proc report*(conf: ConfigRef, node: PNode): TErrorHandling =
   assert node.kind == nkError
   return conf.report(conf.astDiagToLegacyReport(conf, node.diag))
 
+proc fillReportAndHandleVmReport(c: ConfigRef, r: var Report, reportFrom: InstantiationInfo) =
+  r.reportFrom = toReportLineInfo(reportFrom)
+  if r.category in { repSem, repVM } and r.location.isSome():
+    r.context = c.getContext(r.location.get())
+  if r.category == repVM and r.vmReport.trace != nil:
```

Comment:
```suggestion

  if r.category == repVM and r.vmReport.trace != nil:
```
Block-like statements that don't have a strong relation to each other should, for readability, be separated by a new line.